_ml-commons-plugin/model-access-control.md

```diff
@@ -118,8 +121,8 @@ Field |Data type | Description
 :--- | :--- | :---
 `name` | String | The model group name. Required.
 `description` | String | The model group description. Optional.
-`model_access_mode` | String | The access mode for this model. Valid values are `public`, `private`, and `restricted`. When this parameter is set to `restricted`, you must specify either `backend_roles` or `add_all_backend_roles`, but not both. Optional. Default is `restricted`.
-`backend_roles` | Array | A list of the model owner's backend roles to add to the model. Can be specified only if the `model_access_mode` is `restricted`. Cannot be specified at the same time as `add_all_backend_roles`. Optional.
+`access_mode` | String | The access mode for this model. Valid values are `public`, `private`, and `restricted`. When this parameter is set to `restricted`, you must specify either `backend_roles` or `add_all_backend_roles`, but not both. Optional. If you don't specify any of the `access_mode`, `backend_roles`, or `add_all_backend_roles`, the default `access_mode` is `private`.
```

Comment:
```suggestion
`access_mode` | String | The access mode for this model. Valid values are `public`, `private`, and `restricted`. When this parameter is set to `restricted`, you must define a value for either `backend_roles` or `add_all_backend_roles`, but not both. Optional. If `access_mode`, `backend_roles`, and `add_all_backend_roles` are left blank, the default value is `private`.
```

Reply:
You don't leave the fields blank. You omit them.